homeassistant/components/v2c/sensor.py

```diff
@@ -100,6 +116,31 @@ def get_meter_value(value: SlaveCommunicationState) -> str:
         value_fn=lambda evse_data: evse_data.battery_power,
         entity_registry_enabled_default=False,
     ),
+    V2CSensorEntityDescription(
+        key="ssid",
+        translation_key="ssid",
+        icon="mdi:wifi",
```

Comment:
Icon translations

Reply:
the state is the name of the SSID, don't think it makes sense to translate. Signal level would make sense... but I don't know the range :/

Reply:
No I mean, we have a translation_key, we should use `icons.json` to store the icon

Reply:
didn't realise I was already doing that for others 😓 

that's what you get when you seldomly contribute 😅

Reply:
Sorry for the only 2 word comment by the way, should've been more explanatory